src/raftstore/store/metrics.rs

```diff
@@ -75,4 +75,11 @@ lazy_static! {
             "Total nanoseconds spent in raft processing.",
             &["type"]
         ).unwrap();
+
+    pub static ref ENGINE_STATISTICS_GAUGE_VEC: GaugeVec =
```

Comment:
This is the statistics for RocksDB, not for raftstore. I think we should move this to another place, like util/rocksdb